templates/site/js/node.js

```diff
@@ -13,21 +13,15 @@ const converters = {
     },
     "status":(val) =>{
         let node_status = document.createElement("span")
-        const first_letter = val.charAt(0);
-        const first_letter_capitalize = first_letter.toUpperCase();
-        const remaining_word = val.slice(1);
-        const full_word = first_letter_capitalize + remaining_word
+        const full_word = capitalize(val);
```

Comment:
This variable is redundant, just do:

```
node_status.innerText = capitalize(val);
```

Reply:
Removed the step assigning result to full_word.